### PR TITLE
Add kind deployment notes for aesmd pod

### PR DESCRIPTION
**Pull request by astronaut0131**
Polish the document to avoid issue https://github.com/intel/intel-device-plugins-for-kubernetes/issues/1098

### CODE REVIEW

File: cmd/sgx_plugin/README.md [hunk 1]
```diff
@@ -255,6 +255,21 @@ $ kubectl get pods
   intel-sgx-aesmd-mrnm8                1/1     Running   0          3h47m
   sgxdeviceplugin-sample-z5dcq-llwlw   1/1     Running   0          28m
 ```
+> **Note**: If you are using [kind](https://kind.sigs.k8s.io/docs/user/configuration/) to deploy the cluster, you should mount host path`/var/run/aesmd/` to your kind node manually using [Extra Mounts](https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts`). \
```
Comment (mythi) [inline cmd/sgx_plugin/README.md:258]: I'd suggest this clarification that `kind` is just for quick experiments:

```suggestion
> **Note**: For quick experiments, [kind](https://kind.sigs.k8s.io/docs/user/configuration/) can be used to deploy the cluster. With `kind`, host path `/var/run/aesmd/` must be mounted to the nodes manually using [Extra Mounts](https://kind.sigs.k8s.io/docs/user/configuration/#extra-mounts`). \
```

**Comment by mythi** [comment]
@astronaut0131 thanks! The text looks good but please squash the commits to one. Also, perhaps prefix the commit header something like: `sgx: document kind installation`

**Comment by astronaut0131** [comment]
> @astronaut0131 thanks! The text looks good but please squash the commits to one. Also, perhaps prefix the commit header something like: sgx: document kind installation

Done. Thanks for the reminder!

**Review by mythi — APPROVED** [review]

**Review by bart0sh — APPROVED** [review]
/lgtm

**Pull request merged**